crates/biome_deserialize/src/merge.rs

```diff
@@ -0,0 +1,47 @@
+use std::num::{NonZeroU16, NonZeroU32, NonZeroU64, NonZeroU8};
+
+/// Trait that allows deep merging of types, including injection of defaults.
+pub trait Merge {
```

Comment:
Why the trait has not the same name (Mergeable) that the derive macro?

Reply:
No good reason, I've changed the derive to be called `Merge` too!